app/services/prison_visits/api.rb

```diff
@@ -34,7 +34,9 @@ def validate_prisoner(number:, date_of_birth:)
       result.fetch('validation')
     end
 
-    def get_slots(prison_id:, prisoner_number:, prisoner_dob:, use_nomis_slots: false)
+    def get_slots(
```

Comment:
Yeah, me too. I was weak and gave in. Doesn't look like you can disable line length only for method declarations. Do you have a preferred way of formatting?